mozilla-release/mobile/android/base/java/org/mozilla/gecko/preferences/PreferenceManager.java

```diff
@@ -134,4 +138,28 @@ public void setNewsViewExpanded(boolean value) {
         editor.putBoolean(GeckoPreferences.PREFS_CLIQZ_TAB_NEWS_EXPANDED, value).apply();
     }
 
+    public String getSessionId() {
+        return mAppSharedPreferences.getString(SESSION_ID, "");
+    }
+
+    public void setSessionId(String sessionId) {
+        final SharedPreferences.Editor editor = mAppSharedPreferences.edit();
+        editor.putString(SESSION_ID, sessionId).apply();
+    }
+
+    public int getAutoIncrementSequenceNumber() {
+        final int sequenceNumber = mAppSharedPreferences.getInt(TELEMETRY_SEQUENCE, 0);
+        final SharedPreferences.Editor editor = mAppSharedPreferences.edit();
+        editor.putInt(TELEMETRY_SEQUENCE, (sequenceNumber + 1) % 2147483647).apply();
```

Comment:
Is `2147483647` Integer.MAX_VALUE or something else?

Reply:
@jayeshsolanki93 good catch!
@ravjit-cliqz please, no magic numbers.